### PR TITLE
LPS-197326 | Test Fix | Master

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
**Ticket:** [LPS-197326](https://liferay.atlassian.net/browse/LPS-197326)

### CODE REVIEW

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase [hunk 1]
```diff
@@ -494,16 +494,6 @@ definition {
...
-			HeadlessPortalInstanceAPI.addPortalInstance(
-				domain = "www.able.com",
-				portalInstanceId = "www.able.com",
-				virtualHost = "www.able.com");
-
-			User.firstLoginPG(
-				password = "test",
-				userEmailAddress = "test@www.able.com",
-				virtualHostsURL = "http://www.able.com:8080");
-
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase:506]: Hey @Luizcarlosqueiroz, this test was created to cover https://liferay.atlassian.net/browse/LPS-186215 which is reproducible in a new instance. The other changes look good to me!

**Review by patriciajeanperez — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase (hunk 1 above)
Comment (Luizcarlosqueiroz) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase:506]: But considering that the test already creates another instance, is it ok?
I forgot to add this information to the description of the PR.

**Comment by timpak** [comment]
ci:forward

**Pull request closed without merging**